bitnami/nginx/templates/deployment.yaml

```
@@ -133,6 +133,9 @@ spec:
           {{- if .Values.cloneStaticSiteFromGit.gitSync.args }}
           args: {{- include "common.tplvalues.render" (dict "value" .Values.cloneStaticSiteFromGit.gitSync.args "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.syncer.resources }}
+          resources: {{- toYaml .Values.syncer.resources | nindent 12 }}
+          {{- end }}
```

Please rename the values to match the existing settings for the `git-repo-syncer` container
```suggestion
          {{- if .Values.cloneStaticSiteFromGit.gitSync.resources }}
          resources: {{- toYaml .Values.cloneStaticSiteFromGit.gitSync.resources | nindent 12 }}
          {{- end }}
```